.github/workflows/process-load-test-result.yml

```diff
@@ -10,32 +10,43 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
       - name: Process results
-        if: ${{ github.event.client_payload.status == 'success' }}
+        id: processResults
         run: |
           git config --global user.email "ballerina-bot@ballerina.org"
           git config --global user.name "ballerina-bot"
           branch_name="load-test-results"
-          test_scenario="${{ github.event.client_payload.name }}"
           git pull
-          git checkout $branch_name || git checkout -b $branch_name
-          echo ${{ github.event.client_payload.result }} >> load-tests/$test_scenario/results/summary.csv
-          git add load-tests/$test_scenario/results/summary.csv
-          git commit -m "Update $test_scenario test results on $(date)"
+          git checkout -b $branch_name
+          echo '${{ toJson(github.event.client_payload.results) }}' | jq -c '.[]' |
+          while read -r result; do
+            status=$(echo "$result" | jq -r '.status')
+            test_scenario=$(echo "$result" | jq -r '.name')
+            if [[ $status == "success" ]]
+            then
+              echo $(echo "$result" | jq -r '.result') >> load-tests/$test_scenario/results/summary.csv
+              git add load-tests/$test_scenario/results/summary.csv
+              git commit -m "Update $test_scenario test results on $(date)"
+              printf "::set-output name=hasPassed::true"
+            else
+              echo $test_scenario "test has failed with error rate" $(echo "$result" | jq -r '.errorRate')
+              printf "::set-output name=hasFailed::true"
+            fi
+          done
           git push origin "${branch_name}"
         env:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
       - name: Create pull request for new summary
-        if: ${{ github.event.client_payload.status == 'success' }}
+        if: ${{ steps.processResults.outputs.hasPassed }}
         run: gh pr create --title "[Automated] Update summary csv files" --body "Update summary csv files"
```

Comment:
need to update with the latest changes

Reply:
https://github.com/ballerina-platform/module-ballerina-cache/pull/415/files

Reply:
Updated in 252d43b